projects/optic/src/commands/capture/sources/proxy.ts

```diff
@@ -56,19 +63,10 @@ export class ProxyInteractions {
       );
       throw new UserError();
     }
-    if (targetHost.includes('/')) {
-      // accept urls to be passed in rather than pure hosts
-      targetHost = host;
-    }
+    targetHost = host;
 
     const forwardHost = options.mode === 'reverse-proxy' ? origin : targetHost;
 
-    if (protocol)
-      invariant(
-        targetHost.match(/^([a-zA-Z0-9\-]+\.)*[a-zA-Z0-9\-]+(:\d+)?$/),
-        'targetHost must be a valid host (hostname:port)'
-      ); // port optional
-
```

Comment:
again, not sure what this is trying to do since I don't think this would ever latch since `new URL` throws an error on malformed urls